_layouts/rnative-lesson.html

```diff
@@ -15,10 +15,10 @@ <h1 class="title text-2x no-margin">{{page.title}}</h1>
             </p>
             <hr class="divider-rnative" />
             <ul class="list-unstyled text-15x">
-                {% if page.github_link %}
+                {% if page.course_id %}
                     <li>
-                        <a class="rnative-link" href="{{page.github_link}}" target="_blank" rel="noopener noreferrer">
-                            <span class="fa fa-fw fa-chalkboard-teacher"></span> Link to Github
+                        <a class="rnative-link" href="{{site.data.classes[3].github}}/{{page.course_id}}" target="_blank" rel="noopener noreferrer">
```

Comment:
Oh, this is my fault but I should've made classes a dictionary/map, not an array. I'll try to address this in a later PR!

Reply:
If you don't mind, bop a quick `{% comment %} TODO - change this to a map lookup {% endcomment %} ` everywhere you use `site.data.classes[3]`.